jenkinsfile-runner-action/action.yml

```diff
@@ -12,9 +12,16 @@ inputs:
   jcasc:
     description: Jenkins CasC file
     required: false
+  initHook:
+    description: Path to the Groovy init hook file
+    required: false
 runs:
   using: "composite"
   steps:
+    - name: Copy Groovy init hook
+      shell: bash
+      if: ${{inputs.initHook != ''}}
+      run: cp ${{inputs.initHook}} ${JENKINS_ROOT}/jenkins/WEB-INF/groovy.init.d
```

Comment:
This has the potential to overwrite hooks that are a part of this action. Any such attempts should be detected and blocked. Execution of the action should be stopped immediately and a clear error message should be provided.

Reply:
Solved. I add file name conflicts checking.